### PR TITLE
Initial CODEOWNERS for governance repo

### PR DESCRIPTION
**Pull request by vmbrasseur**
Initial list is the TSC as of March 2022 + me.

### CODE REVIEW

**Comment by vmbrasseur** [comment]
@sanfern mentioned on today's TSC call that he's seeing errors while reviewing these. I don't see any sign of errors on this end.

Santhosh, any chance you could give us more information about what errors you're seeing?

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# Refer to https://help.github.com/en/articles/about-code-owners
+
+# Order is important; the last matching pattern takes the most
+# precedence. Try to keep at least two owners per pattern.
+
+# These owners will be the default owners for everything in the repo.
+# Unless a later match takes precedence, they will be requested for
+# review when someone opens a pull request.
+# Initial list: TSC as of March 2022 + VMB 
+* @vmbrasseur @dalalkaran @dthaler @erictice @sanfern
```
Comment (jniesz) [inline CODEOWNERS:11]:
```suggestion
* @vmbrasseur @jniesz @dthaler @erictice @sanfern
```

**Review by jniesz — COMMENTED** [review]
Please replace my name with Karan's and I will approve

**Comment by sanfern** [comment]
> @sanfern mentioned on today's TSC call that he's seeing errors while reviewing these. I don't see any sign of errors on this end.
> 
> Santhosh, any chance you could give us more information about what errors you're seeing?

@vmbrasseur we see following errors in the file:
Unknown owner on line 11: make sure @dthaler exists and has write access to the repository
Unknown owner on line 11: make sure @erictice exists and has write access to the repository

**Comment by sanfern** [comment]
> > @sanfern mentioned on today's TSC call that he's seeing errors while reviewing these. I don't see any sign of errors on this end.
> > Santhosh, any chance you could give us more information about what errors you're seeing?
> 
> @vmbrasseur we see following errors in the file: Unknown owner on line 11: make sure @dthaler exists and has write access to the repository Unknown owner on line 11: make sure @erictice exists and has write access to the repository

I gave them write permissions, they should accept the invite to collaborate.

**Review by dthaler — APPROVED** [review]

**Review by sanfern — APPROVED** [review]
LGTM

**Review by dalalkaran — APPROVED** [review]
LGTM :)

**Comment by dthaler** [comment]
I see Jason requested a change (replace his name with Karan's).  Is that change being made?

**Pull request merged**